TrappingRainWater/main_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include "main.h"
+
+#include <gtest/gtest.h>
+
+class SolutionTest : public ::testing::Test {
+ protected:
+  Solution solution;
+};
+
+TEST_F(SolutionTest, Example1) {
```

Comment:
format your code with clang-format. 
add a .clang-format file to your dir. 
(with this in it: `BasedOnStyle: google`)